decentraland/unity-renderer
# Chore: refactor passport related code to address tech debt

## What does this PR change?

Fix #3961
Refactor passport related code in order to address introduced tech debt

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=chore/refactor-passport-code
2. Verify that the new passport behaves as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
